### PR TITLE
define body background-color

### PR DESCRIPTION
**Pull request by imme-emosol**

### CODE REVIEW

**Comment by alcohol** [comment]
Is there a particular motivation as to why this is necessary?

**Comment by imme-emosol** [comment]
The stylesheet does override the color setting of my browser but not the background-color and dark text is quite hard to read on a dark background.

**Comment by alcohol** [comment]
Thanks

File: web/css/style.css [hunk 1]
```diff
@@ -8,7 +8,7 @@ audio:not([controls]) { display: none; }
 [hidden] { display: none; }
 
 html { font-size: 100%; overflow-y: scroll; -webkit-text-size-adjust: 100%; -ms-text-size-adjust: 100%; }
-body { margin: 0; font-size: 13px; line-height: 1.231; }
```
Comment (wisielec) [inline web/css/style.css]: Duplicate of #

**Pull request merged**